### PR TITLE
check for DC_DOWNLOAD_UNDECIPHERABLE download state

### PR DESCRIPTION
**Pull request by adbenitez**

### CODE REVIEW

File: src/org/thoughtcrime/securesms/ConversationItem.java [hunk 1]
```diff
@@ -390,7 +390,7 @@ else if (text.isEmpty()) {
     }
 
     int downloadState = messageRecord.getDownloadState();
-    if (downloadState != DcMsg.DC_DOWNLOAD_DONE) {
+    if (downloadState != DcMsg.DC_DOWNLOAD_DONE && downloadState != DC_DOWNLOAD_UNDECIPHERABLE) {
```
Comment (adbenitez) [inline src/org/thoughtcrime/securesms/ConversationItem.java:393]: yikes, I was too fast merging, `DcMsg.` in front is missing

Comment (r10s) [inline src/org/thoughtcrime/securesms/ConversationItem.java:393]: for me, a positive list is easier to read,  and also matches better the conditions in the branch.

which one is better in the future, is unclear, however - so feel free to ignore this suggestion :)

```suggestion
    if (downloadState == DcMsg.DC_DOWNLOAD_AVAILABLE || downloadState == DcMsg.DC_DOWNLOAD_FAILURE || downloadState == DcMsg.DC_DOWNLOAD_IN_PROGRESS) {
```

**Review by r10s — APPROVED** [review]

File: src/org/thoughtcrime/securesms/ConversationItem.java (hunk 1 above)
Comment (r10s) [reply src/org/thoughtcrime/securesms/ConversationItem.java:393]: k, too slow :) but no need to change, just forget my suggestion :)

Comment (adbenitez) [reply src/org/thoughtcrime/securesms/ConversationItem.java:393]: I followed your suggestion in a new PR

**Pull request merged**